src/main/java/com/thindeck/life/Lifecycle.java

```diff
@@ -53,6 +54,7 @@
  * @author Yegor Bugayenko (yegor@tpc2.com)
  * @version $Id$
  * @since 0.1
+ * @checkstyle ClassDataAbstractionCoupling (300 lines)
```

Comment:
I disagree. I think that's unnecessary, the class is just 130 lines for now. The warning was introduced by the use of `ServletMfs` due to the deprecation of the static `Manifests.append` class; it does not make the class any less cohesive than before.